src/wx/dialogs/speedup-config.cpp

```diff
@@ -0,0 +1,133 @@
+#include "wx/dialogs/speedup-config.h"
+
+#include <wx/checkbox.h>
+#include <wx/spinctrl.h>
+#include <wx/xrc/xmlres.h>
+
+#include "core/base/check.h"
+#include "wx/config/option-proxy.h"
+#include "wx/dialogs/base-dialog.h"
+
+#include "wx/wxlogdebug.h"
+
+namespace dialogs {
+
+// static
+SpeedupConfig* SpeedupConfig::NewInstance(wxWindow* parent) {
+    VBAM_CHECK(parent);
+    return new SpeedupConfig(parent);
+}
+
+SpeedupConfig::SpeedupConfig(wxWindow* parent)
+    : BaseDialog(parent, "SpeedupConfig") {
+
+    this->Bind(wxEVT_CHECKBOX, std::bind(&SpeedupConfig::ToggleSpeedupFrameSkip, this), XRCID("SpeedupThrottleFrameSkip"));
+    this->Bind(wxEVT_CHECKBOX, std::bind(&SpeedupConfig::ToggleSpeedupMute,      this), XRCID("SpeedupMute"));
+
+    GetValidatedChild<wxCheckBox>("SpeedupMute")->SetValue(OPTION(kPrefSpeedupMute));
+
+    using namespace std::placeholders;
+
+    this->Bind(wxEVT_SPIN_UP,   std::bind(&SpeedupConfig::SetSpeedupThrottle, this, _1), XRCID("SpeedupThrottleSpin"));
+    this->Bind(wxEVT_SPIN_DOWN, std::bind(&SpeedupConfig::SetSpeedupThrottle, this, _1), XRCID("SpeedupThrottleSpin"));
+    this->Bind(wxEVT_SPIN,      std::bind(&SpeedupConfig::SetSpeedupThrottle, this, _1), XRCID("SpeedupThrottleSpin"));
+    this->Bind(wxEVT_TEXT,      std::bind(&SpeedupConfig::SetSpeedupThrottle, this, _1), XRCID("SpeedupThrottleSpin"));
+
+    wxSpinCtrl* speedup_throttle_spin = GetValidatedChild<wxSpinCtrl>("SpeedupThrottleSpin");
+    wxCheckBox* frame_skip_cb         = GetValidatedChild<wxCheckBox>("SpeedupThrottleFrameSkip");
+
+    if (OPTION(kPrefSpeedupFrameSkip) != 0) {
+        speedup_throttle_spin->SetValue(OPTION(kPrefSpeedupFrameSkip) * 100);
```

Comment:
Please, add a comment at the top of this block explaining what `OPTION(kPrefSpeedupFrameSkip)` implies. I am guessing it means speedup frame skip disabled?